html/index.html

```diff
@@ -0,0 +1,223 @@
+              veritatis!.
+            </p>
+          </div>
+
+          <div class="startup-block flex-box">
+            <img class="icons" src="/assets/program2.png" alt="program-icon" />
+            <h2>Building Blocks</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+
+          <div class="growth-hacks flex-box">
+            <img class="icons" src="/assets/program3.png" alt="program-icon" />
+            <h2>Growth Hacks</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+
+          <div class="forum flex-box">
+            <img class="icons" src="/assets/program5.png" alt="program-icon" />
+            <h2>Forum</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+        </div>
+        <div class="button-container">
+          <a href="#">See the whole program</a>
+          <button type="submit" class="program-button">
+            Join CIC Global Summit 2021
+          </button>
+        </div>
+      </div>
+    </section>
+    <div class="section-speakers-heading">
+      <h2>Featured Speakers</h2>
+      <hr />
+    </div>
+
+    <section class="partners-section" id="partners">
+      <h2 class="partners-heading"><strong>Partners</strong></h2>
+      <hr />
+      <div class="partners-flex">
+        <div class="partners-list">
+          <img
+            class="partner-image"
+            src="/assets/partner1.png"
+            alt="our_guest"
+          />
+        </div>
+        <div class="partners-list">
+          <img
+            class="partner-image"
+            src="/assets/partner2.png"
+            alt="our_guest"
+          />
+        </div>
+        <div class="partners-list">
+          <img
+            class="partner-image"
+            src="/assets/partner3.png"
+            alt="our_guest"
+          />
+        </div>
+        <div class="partners-list">
+          <img
+            class="partner-image"
+            src="/assets/partner4.png"
+            alt="our_guest"
+          />
+        </div>
+        <div class="partners-list">
+          <img
+            class="partner-image"
+            src="/assets/partner5.png"
+            alt="our_guest"
+          />
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer-flex">
+        <div class="footer-logo">
+          <a href="index.html"
+            ><img src="/assets/cic_logoReal.png" alt="footer_logo"
+          /></a>
+        </div>
+        <div class="copyright">
+          <p class="copyright">
+            2021 CEOs Community Nigeria
+            <br />
+            SomeRights Reserved
+          </p>
```

Comment:
- [ ] the footer in small mobile view overlaps to the container above. Kindly fix this ⬇️ 

![kl](https://user-images.githubusercontent.com/33067256/129426153-0b8a25b5-affe-429a-af09-8f3bf25ffb4a.png)